MREUnityRuntime/MREUnityRuntimeLib/Core/Physics/JitterBuffer.cs

```diff
@@ -553,13 +591,28 @@ public void Update(ref RigidBodyData entry)
 					entry.MotionType = rb.MotionType;
 					entry.Transform = rb.Transform;
 					entry.Time = _snapshot.Time;
-
+					entry.SetVelocitiesToZero();
```

Comment:
>entry.SetVelocitiesToZero(); [](start = 5, length = 28)

should be else to the following if statement

Reply:
since the if before might be potentially false it is good to have zero initialization

---
In reply to: [453665712](https://github.com/microsoft/mixed-reality-extension-unity/pull/218#discussion_r453665712) [](ancestors = 453665712)